code/lib/core-client/src/executeLoadable.ts

```diff
@@ -2,13 +2,17 @@
 /// <reference types="webpack-env" />
 
 import { logger } from '@storybook/client-logger';
-import type {
-  Path,
-  Store_ModuleExports,
-  Loadable,
-  CoreClient_RequireContext,
-  CoreClient_LoaderFunction,
-} from '@storybook/types';
+import type { Path, ModuleExports } from '@storybook/types';
+
+export interface RequireContext {
+  keys: () => string[];
+  (id: string): any;
+  resolve(id: string): string;
+}
+
+export type LoaderFunction = () => void | any[];
+
+export type Loadable = RequireContext | RequireContext[] | LoaderFunction;
```

Comment:
You moved these types back?

Reply:
They weren't used anywhere else and I think we may even be dropping this `configure()` API, no? (What happened to our plans to do that...?)

Reply:
I think this API drops once, we make storyStoreV7 the only option. I'm ok with moving them.

Reply:
@ndelangen note that the user being able to call `configure(...)` themselves (as opposed to using the `main.js` `stories` field) is not required to use `storiesOf` (you can still use `storiesOf` inside those).

Hmm, it looks like you *already* disabled it anyway? [`363f3f4` (#19553)](https://github.com/storybookjs/storybook/pull/19553/commits/363f3f49c6781d50cc9be682209ab99b5e31091e#diff-43905f965c8574deb664e65c1f5031809241c8f04194676e0da1730e74a59b3a)

Reply:
Yeah I removed it as it was marked deprecated, do you think I was not supposed to?

Reply:
No I do, I'm just confused why we would need the types you can no longer pass anything to `configure` yourself.

Reply:
Hmm not totally sure, I believe I use a LoaderFunction to pass a custom list of requires. Though I'm not sure how it would look if we made changes to fully support the storeV7.

Reply:
Hey @dannyhw we can re-export whichever type you are using in RN so you can continue to support SSv6 in SB7. 

In SSv7, we don't use `configure()` at all so you wouldn't need this for that code path.

Reply:
We'll re-add the export when you've looked at it, until then these types are not being re-exported (this is true before this PR also).